torch_xla/csrc/tensor_impl.cpp

```diff
@@ -74,6 +74,12 @@ c10::intrusive_ptr<c10::TensorImpl> XLATensorImpl::shallow_copy_and_detach()
   return impl;
 }
 
+bool XLATensorImpl::is_contiguous() const {
```

Comment:
When this trigger?
I mean, in which occasions we have seen.
Asking this because their TensorImpl is a moving target, so the less we fiddle with it, the lower the chances we break their CI.

Reply:
Here: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/TensorProperties.cpp#L56. If the tensor is contiguous (always the case for us), the operation is no-op. I doubt this is the kind of method which would see API changes.